server/models/database.js

```diff
@@ -86,4 +89,27 @@ const office = async () => {
     });
 };
 
-export default { pool, users, party, office };
+const candidate = async () => {
+  const candidateTable = `
+  CREATE TABLE IF NOT EXISTS 
+  candidates(
+    candidate_id SERIAL NOT NULL UNIQUE,
+    office INTEGER,
+    party INTEGER,
+    createdBy INTEGER,
+    FOREIGN KEY (party) REFERENCES parties(party_id),
+    FOREIGN KEY (createdBy) REFERENCES users(user_id),
+    FOREIGN KEY (office) REFERENCES offices(office_id),
+    PRIMARY KEY (createdBy, office)
```

Comment:
Will this two primary key ensure that candidate is not created twice?

Reply:
Yes, it will